src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CommunityAPIBookingService.kt

```diff
@@ -81,11 +81,11 @@ class CommunityAPIBookingService(
   }
 
   private fun buildAppointmentRescheduleRequestDTO(appointmentTime: OffsetDateTime, durationInMinutes: Int, npsOfficeCode: String): AppointmentRescheduleRequestDTO {
-    // TODO: CommunityApi and Delius currently don't allow for rescheduling with a different location
     return AppointmentRescheduleRequestDTO(
       updatedAppointmentStart = appointmentTime,
       updatedAppointmentEnd = appointmentTime.plusMinutes(durationInMinutes.toLong()),
       initiatedByServiceProvider = true, // fixme - needs to come from the user - defaulted to SP Initiated Reschedule
+      officeLocationCode = npsOfficeCode
```

Comment:
To clarify, are we intending to send data containing "delius location" or "probation office" codes?

To my knowledge,

- _probation office_ refers to a building, a venue on its own
- _delius location_ refers to the composite ID identifying _probation office_ used by a _provider_

I'd _love_ if we could use the _probation office_ but if we have to use the _location_, could we rename the variable across the board (and optionally create a type alias?), please?

Reply:
(I do believe "office" is the right abstraction here, but it seems the community-api endpoint requires something else?)

Reply:
Would you prefer that we call it `probationOffice` rather than `npsOfficeCode`?

Reply:
I original did a change in an old PR where I used `deliusOfficeLocationCode` for variable name, but I reverted that and  kept the usage of `npsOfficeCode`

I remember the change was quite extensive and touching a lot of classes. 
Given it doesn't change the function, are you okay with this being in another PR?

Reply:
discussion on slack:

David is happy for this to be in a future PR